_maps/shuttles/research_veegee.dmm

```diff
@@ -0,0 +1,128 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"a" = (
+/turf/closed/wall/mineral/titanium,
+/area/shuttle/research)
+"b" = (
+/obj/effect/spawner/structure/window/shuttle,
+/turf/open/floor/plating,
+/area/shuttle/research)
+"c" = (
+/obj/structure/table,
+/turf/open/floor/mineral/titanium/blue,
+/area/shuttle/research)
+"d" = (
+/obj/machinery/computer/shuttle{
+	name = "Research Shuttle Console";
+	possible_destinations = "research_home;research_outpost;landing_zone_dock;mining_public";
+	req_access_txt = "47";
+	shuttleId = "research_veegee"
+	},
+/turf/open/floor/mineral/titanium/blue,
+/area/shuttle/research)
+"e" = (
+/turf/open/floor/mineral/titanium,
+/area/shuttle/research)
+"f" = (
+/obj/structure/chair/comfy/shuttle{
+	dir = 1
+	},
+/turf/open/floor/mineral/titanium/blue,
+/area/shuttle/research)
+"g" = (
+/obj/machinery/light{
+	dir = 8
```

Comment:
doesnt use the directional light subtypes

Reply:
A lot of the things, lights included I believe, didn't have directionals when I started the project. The Research and Mining shuttles are literally just Mining_Box with a different computer at the helm. If that's a major issue, I can go back and start cleaning up all the directionals, though I'm not sure how long that will end up taking.